xml/selinux.xml

```diff
@@ -44,10 +47,10 @@
   <para>
    &selnx; was developed as an additional Linux security solution that uses
    the security framework in the Linux kernel. The purpose was to allow for
-   a more granular security policy that goes beyond what is offered by the
-   default existing permissions of Read, Write, and Execute, and beyond
-   assigning permissions to the different capabilities that are available on
-   Linux. &selnx; does this by trapping all system calls that reach the
+   a more granular security policy that goes beyond the standard 
+   Discretionary Access Controls (DAC), the traditional file permissions
```

Comment:
stop after "traditional file permissions", the list is incomplete and not really helpful.